src/schema/settings.ts

```diff
@@ -80,6 +81,8 @@ export const typeDefs = /* GraphQL */ `
     lastPrompt: String
   }
 
+  ${toGQLEnum(SortCommentsBy, 'SortCommentsBy')}
```

Comment:
Where do we then actually sort by it?

Reply:
On the actual comments request, it is being sort when you pass the right parameter.

Reply:
The feature already exists. The sort is just not persisted at the moment.

Reply:
I was not thinking about the enum specifically (I was just commenting on this part of code). 

I was thinking where do you use this new setting in the comment query, that part is missing?

Reply:
Or do you just add it in this PR and you are gonna hook it up in separate PR?

Reply:
Ah cool. This line was a mistake, it was already imported from the `comments.ts`. So the latest changes removed this part since everything is already integrated. We are just saving it now.

Reply:
> I was thinking where do you use this new setting in the comment query, that part is missing?

It already exists. The query to fetch comments accepts this parameter and sorts accordingly.

Reply:
Ah so you are going to send it from frontend directly, ok